kubetools/deploy/commands/cleanup.py

```diff
@@ -64,19 +64,23 @@ def get_cleanup_objects(build, cleanup_jobs):
             pod_names_already_deleted.add(get_object_name(pod))
 
     namespaces = list_namespaces(build.env)
-    current_namespace = None
     for namespace in namespaces:
         if namespace.metadata.name == build.namespace:
             current_namespace = namespace
+            break
+    else:
+        current_namespace = None
```

Comment:
I think there is an indentation issue here

Reply:
nope

Reply:
Thanks